followthemoney/schema/PositionHeld.yaml

```diff
@@ -0,0 +1,32 @@
+PositionHeld:
```

Comment:
I suspect PositionOccupancy is more correct but it's 7 syllables and I'm a bit worried about our protagonist investigative journalists

Reply:
but it makes the reverse relations a bit clearer:

Position.occupancies
Person.positionsOccupied